docs/scripting-ref.rst

```diff
@@ -331,6 +331,32 @@ Compare:
 See also: :ref:`splash-runjs`, :ref:`splash-jsfunc`,
 :ref:`splash-wait-for-resume`, :ref:`splash-autoload`.
 
+.. _splash-disablejs:
+
+splash:disablejs
```

Comment:
Oh, I see they are mentioned like that in #191, @kmike do you think it's better to have them as separate functions?